src/source/Crypto/Dtls_mbedtls.c

```diff
@@ -8,6 +8,11 @@ mbedtls_ssl_srtp_profile DTLS_SRTP_SUPPORTED_PROFILES[] = {
     MBEDTLS_TLS_SRTP_UNSET,
 };
 
+#if MBEDTLS_VERSION_NUMBER >= 0x03000000
+static mbedtls_ctr_drbg_context ctr_drbg;
```

Comment:
It seems like these were added since we needed them for `mbedtls_pk_check_pair` in `copyCertificateAndKey`, and even though we have these two fields in `struct __DtlsSession`, the `copyCertificateAndKey` doesn't have access to this struct.

Reply:
Maybe an optimization you can make would be to add these two pointer arguments to `copyCertificateAndKey` (since it's only used within the SDK we should be OK for backwards compat.), since these two are already initialized in `createDtlsSession` anyways (avoiding an extra initialization)